### DISCUSSION TITLE
Welcome to Celo!

### DISCUSSION THREAD

**Discussion by lvpeschke** [celo-org/discussions#1] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sodofi** [comment]
Hi 👋 I'm Sophia - I lead devrel at Celo Foundation. Excited to connect with the community on here :)
